app/src/main/java/net/bible/android/view/activity/navigation/History.kt

```diff
@@ -64,13 +66,23 @@ class History : ListActivityBase() {
      */
     protected fun createAdapter(): ListAdapter {
         mHistoryItemList = historyManager.history
-        val historyTextList: MutableList<CharSequence> = ArrayList()
-        for (item in mHistoryItemList!!) {
-            historyTextList.add(item.description)
-        }
-        return ArrayAdapter(this,
+        return object:ArrayAdapter<HistoryItem>(this,
             LIST_ITEM_TYPE,
-            historyTextList)
+            R.id.text1,mHistoryItemList!!){
+            override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+
+                val view = super.getView(position, convertView, parent)
+                val text1 = view.findViewById(R.id.text1) as TextView
```

Comment:
I think you can import using kotlinx synthetic properties, i.e. 
import kotlinx.android.synthetic.main.history_list_item.*
(Android Studio alt-enter auto import works).